cfep-02.md

```diff
@@ -0,0 +1,67 @@
+
+<table>
+<tr><td> Title </td><td> Upgrade default macOS Travis image to `xcode6.4` </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Andreas Kloeckner &lt;inform@tiker.net&gt;</td></tr>
+<tr><td> Created </td><td> Nov 8, 2016</td></tr>
+<tr><td> Updated </td><td> Nov 8, 2016</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda-forge/conda-forge-enhancement-proposals/pull/6 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Upgrade the default Travis image to `xcode6.4`.
+
+## Implementation
+
+This is a one-line change to `.travis.yml`:
+
+```
+osx_image: xcode6.4
+```
+
+*   Adapt `conda smithy` so that it applies that edit on rerender.
+*   Apply the change in the `staged-recipes` master branch.
+*   Rebuild all packages with this? (TBD? Opinions?)
+*   Suggest that all packages include the `toolchain` so that
+    the macOS 10.9 deployment target can be set.
+
+## Rationale
+
+*   The current default image (`beta-xcode6.1`) is deprecated
+    and will cease to be [supported/provided by Travis][travis],
+    so this change will have to happen at some point anyway.
+
+*   We won't be able to compile [clang
+    3.8/3.9](https://github.com/conda-forge/staged-recipes/pull/1481) without
+    this change.
```

Comment:
Also in need of this image are

* [Apache Arrow]( https://github.com/conda-forge/conda-forge.github.io/issues/249#issuecomment-254331475 )
* [`libdynd`]( https://github.com/conda-forge/staged-recipes/pull/1051#issuecomment-233279062 )